### PR TITLE
Add Uchar and Ephemeron in the stdlib module list of the manual

### PR DESCRIPTION
**Pull request by chambart**
Apparently this is infrequent enough that everybody forgot about this. I opened a PR instead of directly pushing to allow someone to provide a better description.

### CODE REVIEW

**Comment by Octachron** [comment]
Since this is a quite unfrequent occurence, it might be nice to add a line in the manual readme file mentioning that stdlib modules should be listed in `library/stdlib.etex`. Maybe something like:

```
##Before release
* Check that `library/stdlib.etex` lists all existing stdlib modules.
```

**Comment by chambart** [comment]
I fear that nobody really reads that readme before the release. I don't know if there is a release checklist anywhere @damiendoligez ?

By the way I also noticed that Sort is part of the list of modules but is deprecated should I also remove it ?

**Comment by damiendoligez** [comment]
There is a release checklist, but this:

> Check that `library/stdlib.etex` lists all existing stdlib modules.

looks like a job for a shell script.

About `Sort`, I don't think we should remove it entirely from the documentation. As long as the module exists, we must document its deprecation.

File: manual/manual/library/stdlib.etex [hunk 1]
```diff
@@ -53,7 +54,8 @@ the above 4 modules \\
 "Buffer" & p.~\pageref{Buffer} & buffers that grow on demand \\
 "Lazy"  & p.~\pageref{Lazy} & delayed evaluation \\
 "Weak" & p.~\pageref{Weak} & references that don't prevent objects
-from being garbage-collected
+from being garbage-collected \\
+"Ephemeron" & p.~\pageref{Ephemeron} & ephemerons and weak hash table
```
Comment (mshinwell) [inline manual/manual/library/stdlib.etex:58]: "tables", plural (same below)

**Comment by damiendoligez** [comment]
Patch applied and script added, in branch 4.03 (commit 5c8baff5955cfc5ac31dce2017d3188a6e86fead) and trunk (commit 393f068a1d4e0b267012b10aac822d11160a7cfd).

**Pull request closed without merging**